### PR TITLE
Implement PoweredDeviceInfo to store consumption infos.

### PR DESCRIPTION
**Pull request by aleemont1**
PoweredInfo to store information about the power consumption of PoweredDevice objects.
This should help reduce the number of parameters for PoweredDevice objects' constructors.

### CODE REVIEW

**Comment by aleemont1** [comment]
@smagnaniuni @michelececcacci Take a look at PoweredDeviceInfo and PoweredDeviceInfoImpl.
THIS MUST NOT BE MERGED BEFORE MERGING #94 AND #95

**Comment by aleemont1** [comment]
@smagnaniuni @michelececcacci waiting for your feedback. I was thinking of removing state, type and ID storing, so PoweredDevices will implement both DeviceInfo and PoweredDeviceInfo.

File: src/main/java/homer/DeviceInfoImpl.java
Comment (smagnaniuni) [inline src/main/java/homer/DeviceInfoImpl.java:1]: Should this be reverted since not necessary anymore?

**Review by smagnaniuni — COMMENTED** [review]
lgtm
There are several typos "istant" -> "instant"

File: src/main/java/homer/DeviceInfoImpl.java
Comment (aleemont1) [reply src/main/java/homer/DeviceInfoImpl.java:1]: Yep, as I said I'll refactor and rebase everything these days.

**Comment by aleemont1** [comment]
Uhm, I think i messed up with git, why do I appear as co-author on those commits?

**Comment by aleemont1** [comment]
Ok, sorry for the mess.
I was thinking #104 #95 #94 can be all merged in one single branch, because I had to do various merges between them, so they should actually be the same. I'll do a rebase and try not to break anything later.

**Comment by michelececcacci** [comment]
ok

**Pull request closed without merging**